.gitignore

```diff
@@ -1,3 +1,5 @@
 /taskcluster-base-test.conf.json
 node_modules/
 .vagrant/
+.test/
+lib/
```

Comment:
Oh, yeah and if we add `.npmignore` it's critical that we ensure all secrets are ignored in `.npmignore` otherwise we could accidentally post secrets on npm :)

Not add npmignore is a simple hack to avoid it... At this stage I'm less sure what is the right thing. It could be adding another footgun, granted this repo has no secrets locally so less of an issue.